MemfaultPackages/bort-ota/src/main/java/com/memfault/bort/ota/UpdateActivity.kt

```diff
@@ -172,6 +174,9 @@ class CheckingForUpdatesFragment : Fragment() {
             progressBar.max = 100
             progressBar.progress = progress
         }
+        if (this::textView.isInitialized) {
+            textView.text = getString(R.string.downloading_format, progress)
```

Comment:
From the logic above, we shoould also write "Downloading", if the progress is negative

Reply:
I don't get what you mean. Checking if the progress is negative before setting it?

Reply:
The progress can be negative in some situations, if we look 5 lines above: `progressBar.isIndeterminate = progress == -1`